lapce-app/src/command.rs

```diff
@@ -746,6 +748,9 @@ pub enum InternalCommand {
         tab_index: usize,
         terminal_index: usize,
     },
+    CallHierarchyIncoming {
+        item: RwSignal<CallHierarchyItemData>,
```

Comment:
It should pass raw data rather than signals in the commands. 

The reason for that is it's easy to mess up with the scope of the signal if doing this way.